datasync/downloader/downloader.go

```diff
@@ -827,6 +859,20 @@ func (d *Downloader) fetchHeaders(p *peerConnection, from uint64, pivot uint64)
 			go p.peer.RequestHeadersByNumber(from, MaxHeaderFetch, 0, false)
 		}
 	}
+	getNextPivot := func() {
+		pivoting = true
+		request = time.Now()
+
+		ttl = d.requestTTL()
+		timeout.Reset(ttl)
+
+		d.pivotLock.RLock()
+		pivot := d.pivotHeader.Number.Uint64()
```

Comment:
Don't we need to check nil here like https://github.com/klaytn/klaytn/pull/1454/files#diff-99163d68e23ed18d94f9b098d41c9d8c23eeb3e71299102c5c25c9a4ef6245e8R897-R899 ?

Reply:
As we talked in f2f, `getNextPivot` is called after checkint `pivot > 0`.